integration-tests/tests/integration/main.rs

```diff
@@ -842,6 +842,113 @@ archive.include = [
         .expect_err("archive should have failed");
 }
 
+#[test]
+fn test_archive_with_build_filter() {
+    set_env_vars();
+
+    let all_test_files = [
+        "basic",
+        "cdylib_example",
+        "cdylib_link",
+        "dylib_test",
+        "my_bench",
+        "nextest_derive",
+        "nextest_tests",
+        "nextest_tests",
+        "other",
+        "other",
+        "proc_macro_test",
+        "segfault",
+        "with_build_script",
+        "wrapper",
+        "nextest_tests",
+        "other",
+    ];
```

Comment:
Can you use the list of fixtures as described here? That way when new fixtures get added this list won't have to be updated. https://github.com/nextest-rs/nextest/blob/main/fixture-data/src/nextest_tests.rs